caffe2/python/onnx/backend.py

```diff
@@ -673,6 +703,21 @@ def optimize_onnx(input, init=False, predict=False):
         out = onnx.optimizer.optimize(input, passes)
         return out
 
+    @classmethod
+    def prepare_zip_archive(cls, file, device='CPU', **kwargs):
```

Comment:
we should probably move this logic (everything but prepare) to onnx.load() instead. But since it's 'quick-and-hacky' - still - maybe just have a standalone function load_zip_archive?